modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/FeaturePropertyTypeBinding.java

```diff
@@ -139,7 +139,6 @@ public Element encode(Object object, Document document, Element value) throws Ex
         if (object instanceof ComplexAttribute) {
             ComplexAttribute complex = (ComplexAttribute) object;
             checkXlinkHref(complex);
-            GML3EncodingUtils.encodeClientProperties(complex, value);
             GML3EncodingUtils.encodeSimpleContent(complex, document, value);
```

Comment:
Wondering about the simple content here... but I guess a feature would never have simple content inside... or would it?